src/Appwrite/Specification/Format/OpenAPI3.php

```diff
@@ -576,6 +576,10 @@ public function parse(): array
                         $type = 'boolean';
                         break;
 
+                    case 'payload':
+                        $type = 'payload';
```

Comment:
We don't use the constant from the `src/Appwrite/Utopia/Response/Model.php` file?

Reply:
Should we? We don't elsewhere in this function